pkg/cloudinitsettings/resources.go

```diff
@@ -0,0 +1,117 @@
+/*
```

Comment:
Why is this not an addon?

Reply:
I'm not sure if this qualifies as an addon, this is more like a pre-requisite for machine-controller to work with OSM.

Reply:
Done